City-of-Pori/pori-kada
# PORI-161 Image alt text required and enable Iframe in wysiwyg.

Image alt text required and enable Iframe in wysiwyg.

To test:
1. drush fra -y
2. go to add images into scald and confirm that alt-text is required.
3. go to edit any page, confirm that there is a possibility to add  an iframe into body field.
